InnerEye/ML/SSL/lightning_containers/ssl_container.py

```diff
@@ -147,22 +147,28 @@ def create_model(self) -> LightningModule:
         # For small images like CIFAR, if using a resnet encoder, switch the first conv layer to a 3x3 kernel instead
         # of a 7x7 conv layer.
         use_7x7_first_conv_in_resnet = False if self.ssl_training_dataset_name.value.startswith("CIFAR") else True
+
+        # Adjust the learning rate to work with multi gpu
+        gpus_per_node = self.num_gpus_per_node()
+        l_rate = self.l_rate * self.num_nodes * (gpus_per_node if gpus_per_node > 0 else 1)
+        print(f"We found {self.num_nodes * gpus_per_node} GPUs, SSL encoder learning rate has been adjusted from {self.l_rate} to {l_rate}")
```

Comment:
About this line:
1. Should it be printed only when `l_rate != self.l_rate`? Otherwise it will pollute the log with a message saying that it has been adjusted from X to X
2. Is it better to use `logging` instead?

Reply:
Yes, for consistency with the rest of the codebase we should use logging.info.
Btw, your output would look funky in the corner case of running on CPU, because then it would say "We found 0 GPUs"